refactor/config_data/test.json

```diff
@@ -15,6 +15,19 @@
         "description": "Sealice per day per cage/farm arriving from wildlife. Note that this value has been determined experimentally in the original code.",
         "value": 150
     },
+    "interfarm_data": {
+        "description": "",
+        "value": {
+            "interfarm_times_path": {
+                "description": "Path to csv file with expected planctonic lice travel time between farms.",
+                "value": "config_data/Fyne_Edays.csv"
```

Comment:
Honestly I'd prefer if config_data was kept in an environment variable and all the paths built from it were relative.